python/mxnet/module/base_module.py

```diff
@@ -700,6 +713,15 @@ def install_monitor(self, mon):
     ################################################################################
     # Computations
     ################################################################################
+    def prepare(self, data_batch):
+        '''Prepare a data batch for forward
```

Comment:
prepare the module for processing a data batch. Usually involves switching bucket and reshaping.